src/functions-reference/positive_lower-bounded_distributions.qmd

```diff
@@ -312,3 +410,19 @@ the upper boundary only. To get the result for the lower boundary, use
 \text{Wiener}(y | \alpha, \tau, 1 - \beta, - \delta)
 \end{equation*}
 For more details, see the appendix of @Vandekerckhove-Wabersich:2014.
+
+### Vectorization
+
+The 5- and 7-argument forms of the `wiener` distribution functions (listed above
+as recieving only `real`) are implemented in such a way where they can be
+fully [vectorized](conventions_for_probability_functions.qmd#prng-vectorization),
+but currently only versions that accept all `real` and all `vector` arguments
+are exposed by Stan. If there are additional signatures that would prove useful,
+please request them on the Stan issue tracker.
+
+### Tolerance tuning
+
+The 5- and 7-argument forms of the `wiener` distribution functions can also
```

Comment:
I worry that people are not going to find this here.

It'd be far better to list it as an optional argument in each function, though we don't have notation to do that.  We need the full signatures for auto-complete, testing, etc. anyway, I think.  Duplicating all the functions in this section with an extra argument and a pointer to the earlier definitions could be helpful for all of this.

Shouldn't this somehow be in the actual definitions?  I guess that requires duplicating them all, pointing to the other implementation and saying the extra parameter controls tolerance.

Reply:
Yes, I wrote it as its own section because we don't have an existing convention for optional arguments and I didn't want to duplicate all the functions just for this, especially when it's not clear how a non-expert would actually chose values for the tolerances